Scale to cloud (#3)
tidymodels/cloudstart: 02_preprocess_with_recipes.Rmd

  mutate_if(is.character, as.factor)
```

> Before moving forward, let's thin our data so we can run these analyses even with minimal (default) computational resources. By doing so we will avoid aborting our RStudio Cloud session.

Reviewer: ```suggestion
Before moving forward, let's reduce the size of our data so we can run these analyses with the default computational resources on RStudio Cloud. By doing so we will avoid aborting our session.
```

Let's sample 20% of the rows and assign it as our data:

```{r}
# Fix the random numbers by setting the seed 
# This enables the analysis to be reproducible when random numbers are used
set.seed(3)

flight_data <- flight_data %>%

>   slice_sample(prop=0.2)

Reviewer: ```suggestion
  slice_sample(prop = 0.2)
```

  mutate(prop = n/sum(n))
```

> For example, the number of `late` and `on_time` flights you get here are less than the number of flights you see in the article. But proportions are very close, suggesting that our random sampling was indeed random and did not over or down-sampled a category vs the other.

Reviewer: ```suggestion
For example, the number of `late` and `on_time` flights you get here are less than the number of flights you see in the article. The proportions are very close, though, suggesting that our random sampling was indeed random and did not over- or under-sample one category vs. the other.
```

```

Prepare the recipe and train the model:

> Be patient, this step will take a little time to compute.

Reviewer: ```suggestion

Be patient; this step will take a little time to compute.
```
